network-api/networkapi/buyersguide/views.py

```diff
@@ -157,7 +157,7 @@ def product_view(request, slug):
         'categories': BuyersGuideProductCategory.objects.all(),
         'product': product_dict,
         'mediaUrl': get_media_url(settings.USE_CLOUDINARY),
-        # 'coralTalkServerUrl': settings.CORAL_TALK_SERVER_URL,
+        'use_commento': settings.USE_COMMENTO,
```

Comment:
_technically_ not needed here, since this file will get deleted when we remove the old PNI app, but it felt reasonable to change anyway just in case.